### DISCUSSION TITLE
Add geth's `debug_traceTransaction`

### DISCUSSION THREAD

**Discussion by mrwillis** [0xPolygon/polygon-edge#280] [Ideas]
### Description

`debug_traceTransaction` is used a lot in different block explorers to trace transactions and provide more metadata on the tx itself. Example:

https://blockscout.com/xdai/mainnet/tx/0x29bc60b639d1f504110637a4f0b5d3e87edc6466797f4332dcd08c5e41677316/internal-transactions

It's a standard geth endpoint and provides similar functionality to parity's trace module. We would be happy to tackle it if given some direction on how to get started!

**Comment by SwaroopH** [comment]
I believe Bor (and Geth) itself don't support trace methods of parity (and now Erigon) but the debug methods are enabled which allows Blockscout to pull meta data. While running edge, we've had to disable that aspect and would be useful to get support for the debug methods at least if not trace.

**Comment by LuxesOwen** [comment]
Hope to support `debug_traceTransaction`, this is very helpful to developers.

**Comment by laviniat1996** [comment]
Hello everyone,

At the moment, we do not support **debug_endpoints**. We will give you an update when we'll add it, but it won't be done in the near future.

**Reply by sebastiendan** [reply]
@laviniat1996 Any updates on this by any chance?
